src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/Details.cshtml

```diff
@@ -11,83 +11,78 @@
 @{
     ViewBag.GaData.Vpv = "/accounts/schemes/details";
 }
+@{Layout = "~/Views/Shared/_Layout_CDN.cshtml"; }
 
-<div class="grid-row">
-    <div class="column-two-thirds">
-        <h1 class="heading-xlarge">
-            <span class="heading-secondary">PAYE scheme</span>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            <span class="govuk-caption-xl"><abbr>PAYE</abbr> scheme</span>
             @Model.Data.EmpRef
-            <span class="heading-secondary">@Model.Data.PayeSchemeName</span>
+            <span class="govuk-caption-xl">@Model.Data.PayeSchemeName</span>
         </h1>
-        
+    </div>
+</div>
 
-        <h2 class="heading-medium">Scheme history</h2>
-        <table>
-            <thead>
-                <tr>
-                    <th scope="col">Date</th>
-                    <th class="numeric" scope="col"><span class="nowrap">English %</span></th>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <table class="govuk-table">
+            <caption class="govuk-table__caption govuk-heading-m">Scheme history</caption>
+            <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th class="govuk-table__header" scope="col">Date</th>
+                    <th class="govuk-table__header" scope="col"><span class="das-no-wrap">English %</span></th>
```

Comment:
Should this have an extra class of 'govuk-table__header--numeric' to right align it?

Reply:
My file doesn't have this class? (.govuk-table__header--numeric)

Reply:
Added.